apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -0,0 +1,125 @@
+        private readonly ComputeFixture _fixture;
+
+        public AcceptanceTest(ComputeFixture fixture, ITestOutputHelper output) =>
+            (_fixture, _output) = (fixture, output);
+
+        [Fact]
+        public void CreateDeleteIPAddress()
+        {
+            string projectId = _fixture.ProjectId;
+            string region = _fixture.Region;
+
+            AddressesClient addressesClient = AddressesClient.Create();
+            var addressName = $"testaddr-csharp-{Guid.NewGuid()}";
+
+            FetchNonExistentAddress();
+            CreateAddress();
+            FetchNewAddress();
+            DeleteAddress();
+
+            void FetchNonExistentAddress()
+            {
+                _output.WriteLine($"Retrieving address with the name: {addressName}");
+                var exc = Assert.Throws<RpcException>(() => addressesClient.Get(projectId, region, addressName));
+                Assert.Equal(StatusCode.NotFound, exc.StatusCode);
+            }
+
+            void CreateAddress()
+            {
+                _output.WriteLine($"Creating address with the name: {addressName}");
+                Address addressResource = new Address
+                {
+                    Name = addressName,
+                    Region = region,
+                    NetworkTier = Address.Types.NetworkTier.Premium
+                };
+
+                Operation insertOperation = addressesClient.Insert(projectId, region, addressResource);
+                _output.WriteLine($"Operation to create address: {insertOperation.Name} status {insertOperation.Status}; start time {insertOperation.StartTime}");
+                insertOperation = PollForCompletion(insertOperation, "create");
+                _output.WriteLine($"Operation to create address completed: status {insertOperation.Status}; start time {insertOperation.StartTime}; end time {insertOperation.EndTime}");
+            }
+
+            void FetchNewAddress()
+            {
+                _output.WriteLine($"Retrieving address with the name: {addressName}");
+                Address readAddr = addressesClient.Get(projectId, region, addressName);
+                Assert.NotNull(readAddr);
+                Assert.Equal(readAddr.Name, addressName);
+            }
+
+            void DeleteAddress()
+            {
+                _output.WriteLine($"Deleting address with the name: {addressName}");
+                Operation deleteOp = addressesClient.Delete(projectId, region, addressName);
+                _output.WriteLine($"Operation to delete address: {deleteOp.Name} status {deleteOp.Status}; start time {deleteOp.StartTime}");
+                deleteOp = PollForCompletion(deleteOp, "delete");
+                _output.WriteLine($"Operation to delete address completed: status {deleteOp.Status}; start time {deleteOp.StartTime}; end time {deleteOp.EndTime}");
+            }
+        }
+
+        private Operation PollForCompletion(Operation operation, string alias)
```

Comment:
Should this be a public extension method in the library, or is this something that will be generated better later?

Reply:
The latter, I believe. At least, that's what I'm hoping.